connector/src/main/scala/com/microsoft/kusto/spark/datasource/KustoRelation.scala

```diff
@@ -130,25 +128,28 @@ private[kusto] case class KustoRelation(kustoCoordinates: KustoCoordinates,
   }
 
   private def getPartitioningColumn: String = {
-    if (readOptions.partitionOptions.column.isDefined) {
-      val requestedColumn = readOptions.partitionOptions.column.get
-      if (!schema.fields.exists(p => p.name equals requestedColumn)) {
-        throw new InvalidParameterException(
-          s"Cannot partition by column '$requestedColumn' since it is not part of the query schema: ${KDSU.NewLine}${schema.mkString(", ")}")
-      }
-      requestedColumn
-    } else schema.head.name
+    readOptions.partitionOptions.column match {
```

Comment:
Just FYI- partitioning is only on debug options and are not presented as user use case ..